include/seeds.guardians.hpp

```diff
@@ -22,6 +28,18 @@ CONTRACT guardians : public contract
 
     ACTION reset();
 
+    ACTION init(name protectable_account, checksum256 invite_hash);
+
+    ACTION join(name guardian_account, checksum256 invite_secret);
```

Comment:
Not sure what the secret is for, or how having a secret would improve things?

- A secret can only be used once - after that it's on chain and no longer a secret
- A secret on the side of the original account would have to be stored somewhere. If they lose their private key they can also lose their secret. If the two are interchangeable because a secret can restore a private key then there's no benefit to having both. 
- If the protected account lost access to their wallet, they will have lost access to the secret as well.
- Same for the guardian accounts - either they have access to their account and secret, or they don't have access to either.

Reply:
Nevermind... secret is to onboard new guardian accts